First deliverable

ToteEmmanuel opened the pull request:
Tiene mockeados los usuarios:
User: John Pass: line
User: Jorge Pass: wize

Seguire trabajando para mejorarlo en el segundo deliverable.

mgranados commented on src/pages/favorites/Favorites.page.jsx:
```diff
@@ -0,0 +1,50 @@
+  const { response: videos, getVideosById } = useYoutube();
+  const [tubeResult, setTubeResult] = useState('');
+
+  useEffect(() => {
```
There's a lot going on here, maybe you could abstract to a Custom Hook.

ToteEmmanuel replied:
moved the card layout logic to a different component.

mgranados reviewed (commented):
## General Notes:
- Filter out the Channels from your query to the YouTube API
- List youtube videos on the `/home` page without requiring a Login
- Mocked Login looks good.
- You could present the additional information on the video details.
- Favorite functionality works good.

### Things to consider:
- Add tests for the components.
- Some console.log present.

ToteEmmanuel commented:
Channel are filtered from responses now.
/home doesn't require a login for search.
added title and description.
added tests to the code.
just left one console.log on an error.

guillermo-rebolledo commented:
Hey 👋  

Just wanted to leave a comment:

- Good use of Hooks, I can see you understood how they work. Always make sure to separate logic into other hooks/components as you see fit.
- Good use of unit tests! 
- Good understanding and good use of styled-components and its APIs!
